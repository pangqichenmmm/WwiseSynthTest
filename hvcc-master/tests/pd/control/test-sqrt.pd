#N canvas 231 101 305 219 10;
#X obj 35 33 loadbang;
#X obj 35 55 t b b b b b b;
#X msg 35 97 -100;
#X msg 73 97 -10;
#X msg 105 97 0;
#X msg 137 97 2;
#X msg 171 97 134;
#X msg 207 97 10000;
#X obj 35 166 print;
#X obj 35 134 sqrt;
#X connect 0 0 1 0;
#X connect 1 0 2 0;
#X connect 1 1 3 0;
#X connect 1 2 4 0;
#X connect 1 3 5 0;
#X connect 1 4 6 0;
#X connect 1 5 7 0;
#X connect 2 0 9 0;
#X connect 3 0 9 0;
#X connect 4 0 9 0;
#X connect 5 0 9 0;
#X connect 6 0 9 0;
#X connect 7 0 9 0;
#X connect 9 0 8 0;
