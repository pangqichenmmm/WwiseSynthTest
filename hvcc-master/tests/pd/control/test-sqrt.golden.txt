[@ 0.000] print: 100
[@ 0.000] print: 11.5758
[@ 0.000] print: 1.41421
[@ 0.000] print: 0
[@ 0.000] print: 0
[@ 0.000] print: 0
